support.sh

```diff
@@ -1,12 +1,23 @@
 #!/usr/bin/env bash
 
+while getopts ":s" opt; do
+  case $opt in
+    s)
+      SSD="true"
+      ;;
+  esac
+done
+
+SSD="${SSD:-false}"
+
 # Required tools
 DOCKER="${DOCKER:-docker}"
 NSENTER="${NSENTER:-nsenter}"
 BRIDGE="${BRIDGE:-bridge}"
 IPTABLES="${IPTABLES:-iptables}"
 IPVSADM="${IPVSADM:-ipvsadm}"
 IP="${IP:-ip}"
+SSDBIN="${SSDBIN:-/ssd}"
```

Comment:
@quadespresso if I drop a `ssd` binary (or shell script calling 'docker run ... ssd') in my $PATH to test, the leading `/` breaks the lookup for the command - I think we can we depend on $PATH inside the container, can we change this to `SSDBIN="${SSDBIN:-ssd}" `

Reply:
https://gist.github.com/adamancini/116ae89b260dc64b67ee46e86ef3a9b1 are my results from support.sh when I call it with `ssd` support

Reply:
LGTM otherwise

Reply:
As discussed on Slack with @adamancini, I'll make the change as discussed above to remove the leading `/`.

Likewise, I'll update `Dockerfile` in `dsinfo` to `/bin` in order to leverage `$PATH` correctly (and to be consistent with a couple other items we're copying to that directory).